library/src/main/java/com/novoda/simplechromecustomtabs/connection/Connection.java

```diff
@@ -14,4 +14,6 @@
 
     void disconnectFrom(@NonNull Activity activity);
 
+    boolean isDisconnected();
```

Comment:
is it too much to add this to the interface? Maybe not when we can Java 8 and have default impl, but could be considered unnecessarily verbose til then